python/python/glide/async_commands/core.py

```diff
@@ -1368,3 +1373,111 @@ async def invoke_script(
                 ["foo", "bar"]
         """
         return await self._execute_script(script.get_hash(), keys, args)
+
+    @dataclass
+    class PubSubMsg:
+        """Describes incoming message
+
+        Attributes:
+            message (str): Incoming message.
+            channel (str): Name of an channel that triggered the message.
+            pattern (Optional[str]): Pattern that triggered the message.
```

Comment:
The channel pattern that triggered the message.

Reply:
How can we have both the channel name and a pattern? shouldn't it be only of of pattern/channel?

Reply:
When subscribed to a pattern, the channel name is being received with new messages?